apps/gitness/src/App.tsx

```diff
@@ -32,7 +33,11 @@ export default function App() {
           children: [
             {
               index: true,
-              element: <>Summary</>
+              element: <RepoSummary />
+            },
+            {
+              path: 'summary',
+              element: <RepoSummary />
```

Comment:
Repository summary is the default page when clicked on repository listing but also can be visited with click on `Summary` tab.